src/styles/blocks/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: inline-block;
+  width: 200px;
+  padding: 32px 16px 16px 16px;
+  box-sizing: border-box;
```

Comment:
move it to the * selector

Reply:
if I move these attributes to the * selector the layout will not match the layout at all

Reply:
Means move `box-sizing` to `*` selector